tests/cypress/e2e/features/shortcuts.js

```diff
@@ -0,0 +1,244 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Customizable Shortcuts', () => {
+    const searchShortcuts = [
+        {
+            title: 'Save the job',
+            description: 'Send all changes of annotations to the server',
+        },
+        {
+            title: 'Switch automatic bordering',
+            description: 'Switch automatic bordering for polygons and polylines during drawing/editing',
+        },
+    ];
```

Comment:
Since it is only used inside one function, I will propose to put it into the function